ivy/functional/frontends/paddle/manipulation.py

```diff
@@ -153,6 +153,9 @@ def squeeze(x, axis=None, name=None):
     return ivy.squeeze(x, axis=axis)
 
 
+@with_supported_dtypes(
```

Comment:
Since these files are not related to the PR, you could add this decorator in the tensor instance of `stack` function.